### PR TITLE
Add option to join to apostrophe-files

### PR DESCRIPTION
**Pull request by plantainrain**
_files here! getcha files here!_  📂 📂  📂

### CODE REVIEW

File: views/widget.html [hunk 1]
```diff
@@ -1,6 +1,8 @@
 {% set widget = data.widget %}
 {% if widget.linkType == 'page' %}
   {% set path = widget._linkPage._url %}
+{% elif widget.linkType == 'file' %}
+  {% set path = widget._linkFile._url%}
```
Comment (grdunn) [inline views/widget.html]: Does `._url` work on an `apostrophe-file` join? Or do you need to use the `apos.attachments()` template helper to pull it out?

Comment (boutell) [reply views/widget.html]: @grdunn It does work, recent feature addition.

Comment (boutell) [inline views/widget.html]: Spacing

**Review by boutell — CHANGES_REQUESTED** [review]

File: views/widget.html (hunk 1 above)
Comment (grdunn) [reply views/widget.html]: 👍

**Review by grdunn — APPROVED** [review]

**Review by boutell — APPROVED** [review]

**Pull request merged**